HelpSource/Guides/WritingClasses.schelp

```diff
@@ -235,92 +390,97 @@ methods should take only one argument to support both ways of expression consist
 
 code::
 MyClass {
-    variable_ { arg newValue;
-        variable = newValue.clip(minval,maxval);
-    }
+
+	variable_ { | newValue |
+		variable = newValue.clip(minval,maxval);
+	}
+
 }
 ::
 
-A setter method should always return the receiver. This is standard OOP practice (outside of SuperCollider as well). Do
-not return the new value from your setter.
+A setter method should always return the receiver. This is standard OOP practice. Never return the new value from your setter.
```

Comment:
(not that you necessarily were the original author, but I feel compelled to say this) - on the contrary, returning the previously held value is sometimes better interface design. The main benefit to returning `this` is the ability to chain, if this guide is going to make ex cathedra pronunciations it should at least provide a rationale :)

Reply:
interesting, yes, I can see that.